src/devtools/client/webconsole/components/FilterBar/FilterBar.js

```diff
@@ -136,11 +140,22 @@ class FilterBar extends Component {
   }
 
   renderClearButton() {
-    const { messagesClearEvaluations } = this.props;
+    const { messagesClearEvaluations, allMessagesById } = this.props;
+    let messagesExist = false;
```

Comment:
Small nit but let's call this `evaluationsExist` since messages refers to all messages.

Reply:
fixed